viewing_party/party.py

```diff
@@ -1,23 +1,189 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
+#DONE
+import re
+
+
+def create_movie(movie_title, genre, rating):
+    dict = {}
+    
+    if movie_title is None:
+        return None
+    elif genre is None:
+        return None
+    elif rating is None:
+        return None
+    else:
+        dict["title"] = movie_title
+        dict["genre"] = genre
+        dict["rating"] = rating
+        return dict
+    pass

```

Comment:
Good work Sujin! We can do the following to DRY up your code and improve readability:

- Condense some of these conditionals into a combined expression
- Build the dictionary in the return statement
- Use a guard clause to exit early the function when falsy values are passed in, reducing the amount of nested code we have and allowing the primary purpose of the function to be slightly easier to read
- And remove `pass` as we no longer need that placeholder when we fill out a function body.

 ```python
def create_movie(title, genre, rating):
    if not title or not genre or not rating:
        return None
        
    return { 
            "title": title, 
            "genre": genre, 
            "rating": rating 
        }
```